contrib/pom.xml

```diff
@@ -487,13 +487,13 @@
     <dependency>
       <groupId>com.datastax.cassandra</groupId>
       <artifactId>cassandra-driver-core</artifactId>
-      <version>2.1.8</version>
+      <version>3.1.0</version>
```

Comment:
https://docs.datastax.com/en/cassandra/3.0/cassandra/features.html states that the cassandra server needs to be upgraded to java 8 for sure. There is not much clarity on the client side requirements. Will add comments after digging into this a bit more.